CHANGES.rst

```diff
@@ -3,6 +3,7 @@
 
 - Expose supported types for import from the dialect
   (`Issue #181 <https://github.com/sqlalchemy-redshift/sqlalchemy-redshift/issues/181>`_)
+- Reflect column comments
```

Comment:
There's no GitHub issue... should I create one?

Reply:
Linking to the PR is fine.